samples/Makefile

```diff
@@ -228,7 +242,15 @@ multdemo: multidemo.o
 ovrldemo: overlaydemo.o
 	$(LD) $(LDFLAGS) -o $@ -C $(SYS)-overlay.cfg -m $@.map $^ $(SYS).lib
 
-OVERLAYLIST := $(foreach I,1 2 3,multdemo.$I ovrldemo.$I)
+OVERLAYLIST := 
+
+ifneq ($(filter ovrldemo,$(EXELIST_$(SYS))),)
+OVERLAYLIST += $(foreach I,1 2 3,ovrldemo.$I)
+endif
+
+ifneq ($(filter multdemo,$(EXELIST_$(SYS))),)
+OVERLAYLIST += $(foreach I,1 2 3,multdemo.$I)
+endif
```

Comment:
Please elaborate on this change. Especially as neither ovrldemo nor multdemo seem to get built for the Plus/4.

Reply:
My mistake; I didn't mean to stage that. I've written an overlay config for the Plus/4 as well, but I wasn't going to put that in this PR.

Reply:
Correction (and let this remind me that I shouldn't work on these things before I've had my coffee...):

Without overlays, this is necessary for the plus/4 use the same D64 recipe as the C64 and C128.